packages/core/platform/platform.js

```diff
@@ -55,6 +55,9 @@ const platforms = [
 	{platform: 'webos', regex: /Web0S;.*Safari\/538.2/, forceVersion: 2},
 	{platform: 'webos', regex: /Web0S;.*Chrome\/38/, forceVersion: 3},
 	{platform: 'webos', regex: /Web0S;.*Chrome\/53/, forceVersion: 4},
+	{platform: 'webos', regex: /Web0S;.*Chrome\/68/, forceVersion: 5},
```

Comment:
Since we saw a wild 'WebOS` (WHY CAPITAL W?!) should we do `/Web[0O]S....`?

Reply:
The zero is indeed odd. :donotwant:. `/[wW]eb[0O]S;...` Maybe we can talk to someone about this...

Reply:
The zero was to differentiate between old PalmHP webOS and LG webOS, so it makes sense historically (and out of our hands anyway).

`/[wW]eb[0O]S;...` does work, but I've since confirmed both 2020 TV and current PR/OSE uses `Web0S`, so I'm not sure where that single report came from.

If we really want to include support for `webOS` in LG to cover the use-case, I'd recommend the easier-to-read:
/(?:webOS|Web0S);.*Chrome\/68/` for webOS 5
/(?:webOS|Web0S);.*Chrome/` for LG webOS catchall